src/python/setup.py

```diff
@@ -177,6 +188,10 @@ def do_not_install(require):
         # See: https://github.com/apple/turicreate/issues/3003
         install_requires.append("tensorflow >= 2.0.0,<= 2.0.1 ")
 
+    # As a hack, the make_wheel script cuts out tensorflow from the above list,
```

Comment:
I'm not really understanding the issue from your comment. Could you explain more? Would it be better to just fix make_wheel?

Reply:
it seems this is from Hoyt. My work continues from his version but ends up in a slightly different way. I will delete this.

Reply:
That's from Hoyt's previous approach. Will be removed.